readme.md

```diff
@@ -50,6 +50,26 @@ github "sindresorhus/Defaults"
 pod 'Defaults'
 ```
 
+## Support types
+
+|    Single Value    |        Array         |            Set            | Dictionary<T: LosslessStringConvertible> |
+| :----------------: | :------------------: | :-----------------------: | :--------------------------------------: |
+| `Int(8/16/32/64)`  | `[Int(8/16/32/64)]`  |  `Set<Int(8/16/32/64)>`   |          `[T: Int(8/16/32/64)]`          |
+| `UInt(8/16/32/64)` | `[UInt(8/16/32/64)]` |  `Set<UInt(8/16/32/64)>`  |         `[T: UInt(8/16/32/64)]`          |
+|      `Double`      |      `[Double]`      |       `Set<Double>`       |              `[T: Double]`               |
+|      `Float`       |      `[Float]`       |       `Set<Float>`        |               `[T: Float]`               |
+|      `String`      |      `[String]`      |       `Set<String>`       |              `[T: String]`               |
+|     `CGFloat`      |     `[CGFloat]`      |      `Set<CGFloat>`       |              `[T: CGFloat]`              |
+|       `Bool`       |       `[Bool]`       |        `Set<Bool>`        |               `[T: Bool]`                |
+|       `Date`       |       `[Date]`       |        `Set<Date>`        |               `[T: Date]`                |
+|       `Data`       |       `[Data]`       |        `Set<Data>`        |               `[T: Data]`                |
+|       `URL`        |       `[URL]`        |        `Set<URL>`         |                `[T: URL]`                |
+| `NSColor` (macOS)  | `[NSColor]` (macOS)  |  `Set<NSColor>` (macOS)   |          `[T: NSColor]` (macOS)          |
+|  `UIColor` (iOS)   |  `[UIColor]` (iOS)   |      `Set<UIColor>`       |           `[T: UIColor]` (iOS)           |
```

Comment:
It feels a bit moot to repeat the basic types we support in each column. I think we can simply say in addition to the single values, we support them wrapped in `Array`, `Set`, `Dictionary`.

This also need to mention support for nested types, for example `[[String: Set<[String: Int]>]]`.